CMakeLists.txt

```diff
@@ -61,9 +61,9 @@ if (NUMA_FOUND)
     list(REMOVE_ITEM CMAKE_REQUIRED_LIBRARIES ${NUMA_LDFLAGS})
 
     set(HAVE_NUMA_H 1)
+  else()
+    message(STATUS "libnuma not found")
```

Comment:
Maybe change this to `libnuma found but disabled by user`

Reply:
I should have addressed this, but double check.